api.md

```diff
@@ -1007,7 +1007,22 @@ AS
         FROM conditions
         GROUP BY time_bucket('1day', timec), location, humidity, temperature;
 ```
+
```

Comment:
Instead add as a tip:  In order to keep the continuous aggregate up to date with the data, the refresh_lag is set to `-<bucket_width>`
e.g. CREATE VIEW continuous_aggregate_view( timec, minl, sumt, sumh )
WITH ( timescaledb.continuous,
    timescaledb.refresh_lag = '-1h',
    timescaledb.refresh_interval = '30m' )
AS
    SELECT time_bucket('1h', timec), min(location), sum(temperature), sum(humidity)
        FROM conditions
        GROUP BY time_bucket('1h', timec), location, humidity, temperature;

Reply:
Ok, added this in as a tip